### DISCUSSION TITLE
希儿希儿希？

### DISCUSSION THREAD

**Discussion by xhxjz** [share121/inter-knot#2292] [其他]
![889F284EE677BF20274BF84481911C54](https://github.com/user-attachments/assets/823f8286-29b4-4c54-82a3-deab22633ce0)

**Comment by KUYUE404** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/43e9d24d154f4da394497f49ee3dfa11.png" alt="希儿比心1" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/95f33cbdff7e4182cf26cec4415cb71d.png" alt="希儿比心2" width="75" >

**Comment by yileinafs** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/07/19/19af38e4ac4fa4a5a985bb2d071c4d13_6639857234667139816.png" alt="希儿_好好好" width="75" >

**Comment by cyq20091029** [comment]
![Uploading 序列 01.00_00_17_38.Still008.png…]()
希儿可爱捏

**Comment by mengsheng521** [comment]
可爱捏
